rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -201,6 +201,8 @@ public class Bridge extends PrecompiledContracts.PrecompiledContract {
     public static final CallTransaction.Function GET_BTC_BLOCKCHAIN_BEST_BLOCK_HEADER = BridgeMethods.GET_BTC_BLOCKCHAIN_BEST_BLOCK_HEADER.getFunction();
     public static final CallTransaction.Function GET_BTC_BLOCKCHAIN_PARENT_BLOCK_HEADER_BY_HASH = BridgeMethods.GET_BTC_BLOCKCHAIN_PARENT_BLOCK_HEADER_BY_HASH.getFunction();
 
+    public static final CallTransaction.Function GET_ACTIVE_POWPEG_REDEEM_SCRIPT = BridgeMethods.GET_ACTIVE_POWPEG_REDEEM_SCRIPT.getFunction();
+
```

Comment:
Where Is This Being Used?

Reply:
Okay Seen

Reply:
<img width="661" alt="image" src="https://user-images.githubusercontent.com/94651730/164724699-928a60e6-a978-4aae-b5a1-dcdaa3866248.png">

It's being used there to decode the results.